content/en/plugins/github-api/use.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Use the GiHub API Plugin
+linkTitle: Use
```

Comment:
`Use` should be `Use Cases`

Reply:
@christosarvanitis I'm leaving the page title and link title as they are. I'm following the style guides I created for Armory docs.